From 5ea6c6b451a56def0a435cfcd68f6c183e3b76df Mon Sep 17 00:00:00 2001
From: Isabel Paredes <isabel.paredes@quantstack.net>
Date: Wed, 16 Apr 2025 18:04:05 +0200
Subject: [PATCH 10/13] Fix extern C error - expected an identifier

The extern "C" construct is for C++ only, but everything here is C code.
---
 src/include/Rmath.h0.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/include/Rmath.h0.in b/src/include/Rmath.h0.in
index 391a9b3046..735bda7be1 100644
--- a/src/include/Rmath.h0.in
+++ b/src/include/Rmath.h0.in
@@ -69,9 +69,7 @@
 
 #if !defined(HAVE_WORKING_LOG1P)
 /* remap to avoid problems with getting the right entry point */
-extern "C" {
 double  Rlog1p(double);
-}
 #define log1p Rlog1p
 #endif
 
